### PR TITLE
Bump hammer katello to 1.5.2

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
Place remove the preleasesource on line 9. That should get the build to pass.

**Review by ekohl — APPROVED** [review]
Let's see if it builds now.

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 1]
```diff
@@ -6,14 +6,13 @@
 %global plugin_name katello
 
 %global release 1
-%global prereleasesource pre.master
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
```
Comment (ekohl) [inline packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec:9]: Looks like this line should also be dropped.

**Comment by parthaa** [comment]
Opened here -> https://github.com/theforeman/foreman-packaging/pull/8164

**Pull request closed without merging**